interface_config.js

```diff
@@ -52,7 +52,7 @@ var interfaceConfig = {
         'livestreaming', 'etherpad', 'sharedvideo', 'settings', 'raisehand',
         'videoquality', 'filmstrip', 'invite', 'feedback', 'stats', 'shortcuts',
         'tileview', 'videobackgroundblur', 'download', 'help', 'mute-everyone',
-        'e2ee'
+        'e2ee', 'security'
```

Comment:
Maybe i'm wrong, but you seem to removed the InfoDialogButton. If i'm right, then it would be nice to remove the related `info` button from here too.

Reply:
Yeah you're right, removed it